### PR TITLE
Deprecate quantum-serverless

### PR DESCRIPTION
**Pull request by Tansito**
### Summary

Part of #1307 

This PR setup the configuration to deprecate quantum-serverless package 🫡

### CODE REVIEW

**Comment by Tansito** [comment]
Let me know what you think about the message!

File: client/README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![Python](https://img.shields.io/badge/Python-3.8%20%7C%203.9%20%7C%203.10-informational)](https://www.python.org/)
 [![Qiskit](https://img.shields.io/badge/Qiskit-%E2%89%A5%200.39.0-6133BD)](https://github.com/Qiskit/qiskit)
 
+# IMPORTANT: THIS PACKAGE WAS DEPRECATED IN FAVOR OF QISKIT-SERVERLESS PACKAGE
+
```
Comment (IceKhan13) [inline client/README.md:9]: let's add link here to new location of repo

Comment (Tansito) [reply client/README.md]: Done [f4857a4](https://github.com/Qiskit/qiskit-serverless/pull/1326/commits/f4857a4764b91296117842ffb10bbf4d6999aa4b)

**Comment by akihikokuroda** [comment]
The `quantum_serverless` directory has been renamed to `qiskit_serverless` in the main branch.  How are the `deprecated` messages look after they merged into the already renamed `main` branch?

**Comment by Tansito** [comment]
This commit is not going to be merged in the `main` branch. It's just going to live here, 0.10.3. Is that what you mean, Aki?

**Comment by akihikokuroda** [comment]
Oh, I see.  Are we going to create the 0.10.3 release and push the python module to pipy?

**Comment by Tansito** [comment]
Yep, just to depecrate it and sunset `quantum-serverless` in favor of `qiskit-serverless`. This PR is basically to setup the message in pypi with the new package and the warning.

**Comment by akihikokuroda** [comment]
I see.  Thanks! How about changing the version to `0.10.2` in [VERSION.txt](client/quantum_serverless/VERSION.txt)?

**Review by akihikokuroda — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**